docs/_templates/layout.html

```diff
@@ -25,6 +25,17 @@
     })(window, document, 'script', 'dataLayer', 'GTM-TXFLBCM');</script>
   <!-- End Google Tag Manager -->
 
+  <!-- Google Tag Manager -->
```

Comment:
Can we add a comment in-line here, to explain why we're doing two GTMs?

(It seems a little weird to duplicate the exact code instead of just passing in the GTM id ('GTM-52GRQSL') into a reusable function, but on reflection this is really short anyways and is an official copy+paste So I think we should keep as-is)

Reply:
this is literally here for a week, possibly less. does it really need commentary?

Reply:
Ah, I thought this was going to be around longer than that. NVM then!